q-and-a.md

```diff
@@ -14,33 +14,36 @@ Nuts is een samenwerkingsverband dat ervoor wil zorgen dat we digitaal kunnen sa
+ICT leveranciers in de zorg staan allemaal voor dezelfde uitdaging: het leggen van een digitaal lijntje met elke andere leverancier. Deze koppelingen leggen we nu vaak één-op-één tussen twee leveranciers, wat kostbaar, tijdrovend en foutgevoelig is. Een softwarepakket onderhoudt al snel tien tot honderd verschillende koppelingen met andere leveranciers. Op basis van de Nuts uitgangspunten zal er een gemeenschappelijke technische basis ontstaan waardoor het voor leveranciers veel makkelijker en goedkoper wordt om een koppeling te realiseren. Daardoor kan er meer energie gestoken worden in belangrijkere zaken, zoals een goede procesondersteuning. Dat resulteert uiteindelijk in een veel betere producten waarmee de zorg beter kan samenwerken.
 
 ## Wat is het verdienmodel?
 
 Nuts is een samenwerkingsverband van softwareleveranciers en heeft daarom ook geen verdienmodel. Door samen te werken aan een gedeelde infrastructuur pakt elke leverancier een deel van de rekening op. Dit laat zich terugbetalen in een betere concurrentiepositie. Immers een betere informatievoorziening is voor de zorgprofessional een belangrijk punt waarop zij hun keuze voor leverancier baseren.
 
 ## Hoe gaat het straks werken voor gebruikers?
 
-Het is de insteek van het MedMij programma dat elke burger inzicht krijgt in zijn of haar eigen zorggegevens via een zelf te kiezen persoonlijke gezondheidsomgeving. Het is de visie van Nuts dat patiënten daarmee op termijn niet alleen inzage krijgen, maar ook zelf kunnen deelnemen in het gesprek met zorgverleners en de controle kunnen nemen over de vraag wie welke gegevens over hem of haar mag inzien.
+Zowel de patiënt als de zorgmedewerker zal niet direct met Nuts werken. De patiënt krijgt een PGO zoals dat nu door MedMij wordt uitgewerkt. Ook de medewerker blijft gewoon zelf een software leverancier kiezen voor de procesondersteuning. Op de achtergrond zullen gegevens tussen deze systemen worden uitgewisseld met behulp van Nuts. Het voordeel hiervan is dat patiënten toestemmingen één keer op uniforme wijze kunnen vastleggen in bijvoorbeeld hun PGO. Zorgverleners kunnen op hun beurt weer inloggen in de diverse systemen met één identiteit. En wat de zorgverlener ook in welk systeem dan ook doet met de data van de patiënt, de patiënt kan dit altijd vanuit het eigen PGO weer inzien. Dat is het voordeel van een universele infrastructuur.
```

Comment:
PGO uitschrijven
MedMij linken
stukje over identiteit weghalen
Welke zorgverlener in welk systeem dan ook 
patient kan inzien en meedoen in de dialoog.